Bottom: ad0041200e5cf8c1b31e9052fe40b992c41f8221
Top:    ad03a8e85b80591b33ba7889abca0a0742b41a85
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 08:24:01 -0500

Fix credential copy to only run on daedalus

Credentials for all hosts end up on daedalus, so limit that play.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index f009bdb9..0727917d 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -112,7 +112,7 @@
     assemble_scripts: true
 
 - name: Cleanup tasks
-  hosts: all
+  hosts: daedalus
   remote_user: ansibleremote
   become: true
   become_user: root
@@ -120,14 +120,7 @@
   tasks:
   - name: Copy credentials to the /etc dir
     copy:
-      src: ./credentials/{{ inventory_hostname }}
-      dest: /etc/ansible-credentials/
-      owner: root
-      group: root
-      mode: 'u=rw,g=,o='
-  - name: Copy build system credentials to /etc dir
-    copy:
-      src: ./credentials/buildsystem.novatech-llc.com
+      src: "{{ playbook_dir }}/credentials/"
       dest: /etc/ansible-credentials/
       owner: root
       group: root
